lib/web/integrations_awsoidc.go

```diff
@@ -213,11 +213,11 @@ func (h *Handler) awsOIDCConfigureDeployServiceIAM(w http.ResponseWriter, r *htt
 	// teleport integration configure deployservice-iam
 	argsList := []string{
 		"integration", "configure", "deployservice-iam",
-		fmt.Sprintf(`--cluster="%s"`, clusterName),
-		fmt.Sprintf(`--name="%s"`, integrationName),
-		fmt.Sprintf(`--aws-region="%s"`, awsRegion),
-		fmt.Sprintf(`--role="%s"`, role),
-		fmt.Sprintf(`--task-role="%s"`, taskRole),
+		fmt.Sprintf("--cluster=%s", clusterName),
```

Comment:
What with spaces? `""` allows the string to have spaces or any other white characters. Removing it will make it fail on this case.

Reply:
Sorry, it was not clear to me
Is there a case you have in mind that would break the concatenation of fields?

Reply:
White characters are not valid for those identifiers (AWS IAM roles, or Teleport resources).